projects/valtimo/config/assets/core/nl.json

```diff
@@ -160,6 +160,12 @@
       "cancel": "Annuleren",
       "delete": "Verwijderen"
     },
+    "delete": {
+      "delete": "Dossier verwijderen",
+      "title": "Dossier verwijderen",
```

Comment:
You don't need both of these, please remove one

Reply:
They are 2 different items on the page though that you might want to change independently (1 is for the button, the other one for the modal header). I'm not sure what you usually do in this kind of situation. Are you sure you want me to remove one?

Reply:
As long as they have the same texdt, I should we should just stick with one